OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -520,8 +484,8 @@ bool HasAdequateAirUnitReloadBuildings(ActorInfo actorInfo)
 			var hasAmmoPool = actorInfo.TraitInfos<AmmoPoolInfo>().Any();
 			if (hasAmmoPool && aircraftInfo.RearmBuildings.Count > 0)
 			{
-				var countOwnAir = CountUnits(actorInfo.Name, Player);
-				var countBuildings = aircraftInfo.RearmBuildings.Sum(b => CountBuilding(b, Player));
+				var countOwnAir = CountActorsWithTrait<IPositionable>(actorInfo.Name, Player);
+				var countBuildings = aircraftInfo.RearmBuildings.Sum(b => CountActorsWithTrait<Buildable>(b, Player));
```

Comment:
This should be `Building` instead of `Buildable`, but I'll fix it myself so we can merge this quickly.